canvasapi/module.py

```diff
@@ -126,9 +126,10 @@ def create_module_item(self, module_item, **kwargs):
             if "content_id" in module_item:
                 kwargs["module_item"] = module_item
             else:
-                raise RequiredFieldMissing(
-                    "Dictionary with key 'content_id' is required."
-                )
+                if "type" in module_item != ("ExternalUrl" or "Page" or "SubHeader"):
+                    raise RequiredFieldMissing(
+                        "Dictionary with key 'content_id' is required."
+                    )
```

Comment:
What's up with this change?

Reply:
an attempt at solving #350, not working though 👎🏾

Reply:
Gonna remove this from this branch/PR.